Docs/source/creating_a_problem.rst

```diff
@@ -58,12 +54,10 @@ Here:
 * `default` is the default value of the runtime parameter.  It may be
   overridden at runtime by reading from the namelist.
 
-* `namelist` indicates if the variable should be in the namelist and
-  controlled at runtime.  The namelist column should have a ``y`` if
-  you want the parameter to be read from the ``&fortin`` namelist in
-  the ``probin`` file at runtime.  If it is empty or marked with
-  ``n``, then the variable will still be put into the
-  ``probdata_module`` but it will not be initialized via the namelist.
+* `namelist` indicates if the variable should be able to be set as
+  a runtime parameter.  If it is empty or marked with
+  ``n``, then the variable will still be creates in the ``problem`` namespace,
+  but it will not be able to be set via the commandline or inputs file.
```

Comment:
We should probably add a sentence here to explain why you'd want to use this (i.e. for global variables you want to set at runtime based on some combination of the other parameters or other information about the problem setup).

Reply:
done